Singer.API/Controllers/SingerLocationController.cs

```diff
@@ -0,0 +1,54 @@
+using Microsoft.AspNetCore.Authorization;
+using Singer.Models;
+using Singer.DTOs;
+using Microsoft.AspNetCore.Mvc;
+using Singer.Services.Interfaces;
+using Microsoft.AspNetCore.Http;
+using System.Threading.Tasks;
+
+namespace Singer.Controllers
+{
+   [Route("api/[controller]")]
+   [Authorize]
+   public class SingerLocationController : DataControllerBase<SingerLocation, SingerLocationDTO, CreateSingerLocationDTO, UpdateSingerLocationDTO>
```

Comment:
Ah, hah. This isn't this then `api/singerlocations` instead of `api/eventlocations`? (See comment above)